src/Controller/ConfigController.php

```diff
@@ -262,11 +262,15 @@ public function getAction(Request $request): JsonResponse
         $name = $request->get('name');
 
         $configuration = Dao::getByName($name);
+
         if (empty($configuration)) {
             throw new Exception('Name does not exist.');
         }
 
-        if ($configuration && isset($configuration->configuration->general->executor)) {
+        if (
+            isset($configuration->configuration->general->executor) &&
+            !empty($configuration->configuration->general->executor)
```

Comment:
better use class_exists?

Reply:
Also using empty and isset is a little redundant, you could just use !empty && class_exists